internal/agent/device/lifecycle/manager.go

```diff
@@ -11,8 +11,9 @@ import (
 	"github.com/flightctl/flightctl/internal/agent/client"
 	"github.com/flightctl/flightctl/internal/agent/device/errors"
 	"github.com/flightctl/flightctl/internal/agent/device/fileio"
-	"github.com/flightctl/flightctl/internal/agent/device/status"
+	fcstatus "github.com/flightctl/flightctl/internal/agent/device/status"
```

Comment:
just curious why the prefix?

Reply:
There was some confusion around `status` as a module vs. `status` as a variable - let me see if it was my confusion vs. the compiler's, though. If it works without the prefix I can remove it.